gen/pb-python/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "sigstore-protobuf-specs"
+version = "0.0.1"
```

Comment:
Note to self (see https://github.com/sigstore/protobuf-specs/issues/22): at some point we're going to want one canonical source of truth for the version number. Or at least a "bump version script" that will update this.

(You don't need to worry for this PR)

Reply:
+1 -- in the past I've done this with a top-level `VERSION` file, although the downside to that is that it doesn't play nicely with `pyproject.toml`.

Reply:
(We could also keep it synced automatically with a GitHub Action).

Reply:
Created #26 to track this.